app/contestants/page.tsx

```diff
@@ -1,9 +1,9 @@
 import { getTeamList, ITeam } from "../utils/wikiQuery"
-import { wikiUrl, getWikipediaContestantData } from "../utils/wikiFetch"
+import { wikiUrl, apiUrl, getWikipediaContestantData } from "../utils/wikiFetch"
 
 export default async function Contestants() {
 
-    const wikiContestants = await getWikipediaContestantData()
+    const wikiContestants = await getWikipediaContestantData(apiUrl)
     const final = getTeamList(wikiContestants)
```

Comment:
You could convert these two lines into a util function called something like "getPageData"

Reply:
This is a decent idea, but I am not yet convinced that now is the right time to do that. [see Rule Of Three](https://en.wikipedia.org/wiki/Rule_of_three_(computer_programming))